ci/README.md

```diff
@@ -1,6 +1,6 @@
 # Continuous integration / analysis directory
 
-[![Build Status](https://travis-ci.org/greenelab/manubot-rootstock.svg?branch=master)](https://travis-ci.org/greenelab/manubot-rootstock)
+[![Build Status](https://travis-ci.com/manubot/rootstock.svg?branch=master)](https://travis-ci.com/manubot/rootstock)
```

Comment:
Ah this is an issue. SETUP instructions never update this badge to refer to the manubot instance compared to rootstock. Something to fix in the future.